pycbc/workflow/jobsetup.py

```diff
@@ -627,11 +627,18 @@ class PyCBCInspiralExecutable(Executable):
     file_input_options = ['--gating-file']
 
     def __init__(self, cp, exe_name, ifo=None, out_dir=None,
-                 injection_file=None, tags=None):
+                 injection_file=None, tags=None, reuse_executable=False):
         if tags is None:
             tags = []
-        super(PyCBCInspiralExecutable, self).__init__(cp, exe_name, None, ifo,
-                                                      out_dir, tags=tags)
+        super(PyCBCInspiralExecutable, self).__init__(
+            cp,
```

Comment:
Why one option per line? Won't hold up PR for this, but I don't really like this style.

Reply:
I'm following: https://github.com/psf/black

.. At some point I might just run this on our module code.